### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by natalia-matash**
- [DEMO LINK](https://natalia-matash.github.io/layout_search-bar-airbnb/)
- [TEST REPORT LINK](https://natalia-matash.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,16 +1,38 @@
...
-    <h1>Search bar airbnb</h1>
+    <form
+      action="#"
+      method="post"
+      class="bigBar"
+      data-qa="big"
+    >
+      <input
+        type="text"
+        class="input input_big"
```
Comment (agniya-i) [inline src/index.html:20]:
```suggestion
        class="input"
```

File: src/index.html [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+    </form>
+    <form
+      action="#"
+      method="post"
+      class="smallBar"
+      data-qa="small"
+    >
+      <input
+        type="text"
+        class="input input_small"
```
Comment (agniya-i) [inline src/index.html:33]:
```suggestion
        class="input input--small"
```

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+  font-weight: 300;
+  width: 100%;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  color: #3d4e61;
+}
+
+.input_big {
```
Comment (agniya-i) [inline src/style.css:34]: move styles from this class to `.input`, leave only `input--small` to override values for small input (look into checklist p2)

**Review by agniya-i — CHANGES_REQUESTED** [review]

**Comment by SanyaBratashchuk** [comment]
Why did you close PR?

**Comment by natalia-matash** [comment]
Because I couldn't change it like in instruction. I saw that PR was
rejected, after I corected mistakes and in MA app PR was still rejected. I
reopened PR, still rejected. So I create new PR.

понедельник, 26 декабря 2022 г. пользователь Sasha Bratashchuk <
***@***.***> написал:

**Pull request closed without merging**